makefile.shade

```diff
@@ -2,6 +2,7 @@
 var VERSION='0.1'
 var FULL_VERSION='0.1'
 var AUTHORS='Microsoft Open Technologies, Inc.'
+-BuildQuality = "";
```

Comment:
This declares the BuildQuality for the entire repo. Since the two projects in this repo have different build quality, this resets it.